web/src/portico/tabbed-instructions.ts

```diff
@@ -56,7 +58,11 @@ export function activate_correct_tab($codeSection) {
     if (!$active_list_items.length) {
         $li.first().addClass("active");
         const language = $li.first()[0].dataset.language;
-        $blocks.filter("[data-language=" + language + "]").addClass("active");
+        if (language) {
+            $blocks.filter("[data-language=" + language + "]").addClass("active");
+        } else {
+            throw new Error("Language Not Found in the Dataset");
```

Comment:
We likely do not want to throw an error in production. Let's call `blueslip.error` here so that we can catch it only in development.

Reply:
Yeah, you are right, working on it.